catalog/transformer/catalog-transformer-tika-input/src/main/java/ddf/catalog/transformer/input/tika/TikaInputTransformer.java

```diff
@@ -391,11 +391,12 @@ public Metacard transform(InputStream input, String id)
                 metadata, id, metadataText, metacardType, useResourceTitleAsTitle);
         if (StringUtils.isNotBlank(bodyText)) {
           metacard.setAttribute(new AttributeImpl(Extracted.EXTRACTED_TEXT, bodyText));
+          processContentMetadataExtractors(bodyText, metacard);
         }
 
-        processContentMetadataExtractors(bodyText, metacard);
-
-        processMetadataExtractors(bodyText, metacard);
+        if (StringUtils.isNotBlank(metadataText)) {
+          processMetadataExtractors(metadataText, metacard);
```

Comment:
Should we let the `MetadataExtractor`s decide whether or not they want to process this input instead of skipping this completely if `metadataText` is empty?

Reply:
The interface doesn't guarantee that they have to handle a blank/null case. If it did, we could remove these checks.